app/posts/2021-10-23-end-to-end-application-journey-round-3.md

```diff
@@ -0,0 +1,295 @@
+
+>“If we know that someone's good, we are thinking, are they interviewing elsewhere? and how quickly do we need to get them in before another school hires them.”
+
+>"So that tells me we have invited her to an interview, it doesn’t tell me if she has accepted or if she is not going to attend, has she sent me a message back?"
+
+6. We tested a feature where hiring staff were able to either download all of the applications at once or share the applications directly to an email address.
+
+{{ appFigure({
+  image: {
+    file: "share-1.png",
+    alt: "initial step of the share flow"
+  }
+}) }}
+
+{{ appFigure({
+  image: {
+    file: "share-4.png",
+    alt: "success banner once applications have been shared"
+  }
+}) }}
+
+We found that the role of administrative staff is purely to pass all of the applications on to the panel. More senior hiring staff (HR managers of business managers for example) may download and review the applications before passing them on. Having the option to both download or share all of the applications simplified this process.
+
+This also highlighted the varying levels of responsibility amongst hiring staff and potential need for more tailored user experiences.
+
+7. Schools are heavily reliant on having hard copies of applications to assist them with the shortlisting process. This is mainly for speed, as applications are compared side by side and notes are made directly onto the paper applications. Hard copies are stored in files, which they seem reluctant to discard.
+
+We found our current download does not fully support the needs of the SLT as it’s format is not optimal for comparing and quickly reviewing a candidate's information.
+
+8. We tested a feature where hiring staff were able to make notes within an application. These notes would be seen by other members of the recruitment team.
+
+{{ appFigure({
+  image: {
+    file: "notes-1.png",
+    alt: "the note tab, within the application screen"
+  }
+}) }}
+
+{{ appFigure({
+  image: {
+    file: "notes-3.png",
+    alt: "initial step of the add a note flow"
+  }
+}) }}
+
+We found notes are made by both hiring staff and the SLT both before and after shortlisting has taken place. Sticky notes, handwritten notes and  highlighted text communicate red flags such as issues around references, right to work in the UK, if that candidate had appeared rude during any contact that had been made or to mark areas of the application that seemed incomplete.
```

Comment:
Personally I'd go whether instead of if here, and add a comma before the last or? It's a long list!